### ISSUE TITLE
Add MCMC sampler

### ISSUE THREAD

**Issue by andreufont** [igmhub/picca#516]
I'm trying to fit eBOSS data with / without metals, with / without HCDs, trying different models for HCDs... and I think it would be quite useful to be able to study degeneracies with a sampler that produces contour plots. 

The current fitter reports a correlation matrix at the end, but the numbers there look suspicious (are beta and bias correlated at the 0.94 level???), and even if the matrix is correct it would be interesting to see the contours / banana plots. 

How difficult would it be to implement?

**Comment by andreicuceu** [comment]
Here

**Comment by andreufont** [comment]
Hi @andreicuceu , can you reassign this ticket to yourself?

**Comment by andreufont** [comment]
@londumas , can you assign the issue to @andreicuceu ?

**Comment by londumas** [comment]
I just invited @andreicuceu to github, as soon as he says yes. I will be able to assign it to him.

**Comment by ngbusca** [comment]
@andreufont @andreicuceu check out the existing mcmc sampler `picca_mcmc`, there might be something useful there....

**Comment by andreufont** [comment]
Yes, @ngbusca , Andrei is aware of it. He will probably start something from scratch (using polychord as sampler?), but will first take a good look at the existing sampler. Thanks!

**Comment by iprafols** [comment]
I believe this was solved in Vega. @andreicuceu any comments? @andreufont do you agree?  If you do, maybe we should close the issue

**Comment by andreicuceu** [comment]
Hi @iprafols, this is indeed addressed by Vega and was also implemented in fitter2 by https://github.com/igmhub/picca/pull/686. So we can close this issue.
